docs/developers/troubleshooting.rst

```diff
@@ -0,0 +1,29 @@
+===============
+Troubleshooting
+===============
+
+Docker Installations
+--------------------
+
+.. _Windows Docker Installation:
+
+Windows
+^^^^^^^
+
+If changes to JS files are not being watched or hot reloaded by the host file system, follow 
```

Comment:
In general , it is a bad idea to keep files on the Windows system if you are going to interact with them from WSL2. As it is a performance killer, I think you can just merge this advice into the other windows and say something like You should clone the files in the WSL2 filesystem otherwise code watchers might not work.

Reply:
Yeah I agree. Having this information on the development server setup page will save Windows developers a lot of trouble.